src/components/signup/SignupSvg.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import commonStyle from '../common/common.scss';
+
+export default function SignupSvg() {
+  return (
+    <div className="msg-svg" style={{ commonStyle }}>
+      <div className="msg-svg__message">
+        <p>If you already have an account!</p>
+      </div>
+      <div className="msg-svg__Login-svg">
+        <a href="/login">
+          <svg
```

Comment:
it will be a good idea to put the svg in its separate file.

Reply:
Put svg files in assets and import them from there

Reply:
Fixed